tests/dvslib/dvs_vlan.py

```diff
@@ -15,7 +15,7 @@ def create_vlan(self, vlan):
 
     def create_vlan_hostif(self, vlan, hostif_name):
```

Comment:
do we want to change this hostif_name too?

Reply:
Not modifying the param names but just the field name